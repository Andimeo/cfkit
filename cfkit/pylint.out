************* Module cfkit.commands
commands.py:186:16: C0303: Trailing whitespace (trailing-whitespace)
commands.py:187:0: C0303: Trailing whitespace (trailing-whitespace)
commands.py:194:0: C0301: Line too long (122/100) (line-too-long)
commands.py:201:0: C0301: Line too long (126/100) (line-too-long)
commands.py:204:0: C0304: Final newline missing (missing-final-newline)
commands.py:1:0: C0114: Missing module docstring (missing-module-docstring)
commands.py:8:0: C0116: Missing function or method docstring (missing-function-docstring)
commands.py:8:19: C0103: Argument name "x" doesn't conform to snake_case naming style (invalid-name)
commands.py:26:0: C0116: Missing function or method docstring (missing-function-docstring)
commands.py:26:31: C0103: Argument name "inputPath" doesn't conform to snake_case naming style (invalid-name)
commands.py:26:42: C0103: Argument name "outputPath" doesn't conform to snake_case naming style (invalid-name)
commands.py:27:13: C3001: Lambda expression assigned to a variable. Define a function using the "def" keyword instead. (unnecessary-lambda-assignment)
commands.py:27:29: W1510: 'subprocess.run' used without explicitly defining the value for 'check'. (subprocess-run-check)
commands.py:28:28: C0103: Argument name "x" doesn't conform to snake_case naming style (invalid-name)
commands.py:35:32: C0103: Argument name "L" doesn't conform to snake_case naming style (invalid-name)
commands.py:37:4: C0103: Variable name "ok" doesn't conform to snake_case naming style (invalid-name)
commands.py:38:4: C0103: Variable name "R" doesn't conform to snake_case naming style (invalid-name)
commands.py:49:8: C0103: Variable name "ok" doesn't conform to snake_case naming style (invalid-name)
commands.py:113:6: C0103: Variable name "l" doesn't conform to snake_case naming style (invalid-name)
commands.py:115:34: C0103: Argument name "K" doesn't conform to snake_case naming style (invalid-name)
commands.py:117:28: C0103: Argument name "x" doesn't conform to snake_case naming style (invalid-name)
commands.py:120:17: C3001: Lambda expression assigned to a variable. Define a function using the "def" keyword instead. (unnecessary-lambda-assignment)
commands.py:137:12: C0103: Variable name "c" doesn't conform to snake_case naming style (invalid-name)
commands.py:141:14: C0103: Variable name "c" doesn't conform to snake_case naming style (invalid-name)
commands.py:161:8: C0103: Variable name "K" doesn't conform to snake_case naming style (invalid-name)
commands.py:162:12: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
commands.py:44:2: R0912: Too many branches (27/12) (too-many-branches)
commands.py:44:2: R0915: Too many statements (79/50) (too-many-statements)
commands.py:184:7: E0601: Using variable 'lang' before assignment (used-before-assignment)
commands.py:26:0: R0915: Too many statements (96/50) (too-many-statements)

------------------------------------------------------------------
Your code has been rated at 7.34/10 (previous run: 0.00/10, +7.34)

