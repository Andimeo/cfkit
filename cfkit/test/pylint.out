************* Module test
test.py:188:0: C0301: Line too long (122/100) (line-too-long)
test.py:1:0: C0114: Missing module docstring (missing-module-docstring)
test.py:7:0: C0116: Missing function or method docstring (missing-function-docstring)
test.py:7:19: C0103: Argument name "x" doesn't conform to snake_case naming style (invalid-name)
test.py:27:0: C0116: Missing function or method docstring (missing-function-docstring)
test.py:27:31: C0103: Argument name "inputPath" doesn't conform to snake_case naming style (invalid-name)
test.py:27:42: C0103: Argument name "outputPath" doesn't conform to snake_case naming style (invalid-name)
test.py:28:13: C3001: Lambda expression assigned to a variable. Define a function using the "def" keyword instead. (unnecessary-lambda-assignment)
test.py:28:29: W1510: 'subprocess.run' used without explicitly defining the value for 'check'. (subprocess-run-check)
test.py:30:26: C3001: Lambda expression assigned to a variable. Define a function using the "def" keyword instead. (unnecessary-lambda-assignment)
test.py:33:32: C0103: Argument name "L" doesn't conform to snake_case naming style (invalid-name)
test.py:35:4: C0103: Variable name "ok" doesn't conform to snake_case naming style (invalid-name)
test.py:36:4: C0103: Variable name "R" doesn't conform to snake_case naming style (invalid-name)
test.py:47:8: C0103: Variable name "ok" doesn't conform to snake_case naming style (invalid-name)
test.py:111:6: C0103: Variable name "l" doesn't conform to snake_case naming style (invalid-name)
test.py:113:34: C0103: Argument name "K" doesn't conform to snake_case naming style (invalid-name)
test.py:115:26: C3001: Lambda expression assigned to a variable. Define a function using the "def" keyword instead. (unnecessary-lambda-assignment)
test.py:117:17: C3001: Lambda expression assigned to a variable. Define a function using the "def" keyword instead. (unnecessary-lambda-assignment)
test.py:134:12: C0103: Variable name "c" doesn't conform to snake_case naming style (invalid-name)
test.py:138:14: C0103: Variable name "c" doesn't conform to snake_case naming style (invalid-name)
test.py:158:8: C0103: Variable name "K" doesn't conform to snake_case naming style (invalid-name)
test.py:159:12: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
test.py:42:2: R0912: Too many branches (27/12) (too-many-branches)
test.py:42:2: R0915: Too many statements (80/50) (too-many-statements)
test.py:27:0: R0915: Too many statements (96/50) (too-many-statements)

------------------------------------------------------------------
Your code has been rated at 7.92/10 (previous run: 3.84/10, +4.08)

